File: 02-Python/07-who-is-buying-lunch.py
#!/usr/bin/python

import random;

your_input = input("Enter names separated by space: \n")

names = your_input.split()

def whos_paying(names):
    random_num = random.randint(0, len(names)-1)

    random_name = names[random_num]

    print(random_name + " is going to buy lunch today!")


whos_paying(names)

File: 01-JavaScript/07-who-is-buying-lunch.js
const prompt = require("prompt-sync")({ sigint: true });

const yourInput = prompt("Enter names separated by space: ");

const names = yourInput.split(" ");
console.log(names.length);

function whosPaying(names) {

    const random = Math.floor(Math.random() * names.length);

    const randomName = names[random];

    console.log(randomName + " is going to buy lunch today!")
}

whosPaying(names);